photon-server/build.gradle

```diff
@@ -144,6 +172,18 @@ spotless {
     }
 }
 
+run {
+    if (project.hasProperty("profile")) {
+        jvmArgs=[
+                "-Dcom.sun.management.jmxremote=true",
+                "-Dcom.sun.management.jmxremote.ssl=false",
+                "-Dcom.sun.management.jmxremote.authenticate=false",
+                "-Dcom.sun.management.jmxremote.port=5000",
+                "-Djava.rmi.server.hostname=192.168.1.96",
```

Comment:
Remove?

Reply:
I changed this to use `0.0.0.0` so that we can keep this flag (I think it's pretty useful and it would be nice to keep it around.)